docs/robots/husky/maintenance_husky.mdx

```diff
@@ -23,6 +74,90 @@ import ComponentUpdatingPumaFirmware from "/components/maintenance/firmware_upda
+The bulk of the capacity (70%) is regained during the first stage.
+
+It is important to let the charger complete all three stages whenever possible, to help maintain the life of the battery.
+The initial stage will take 5-7 hours, and a complete charge may take up to 14 hours.
+
+<center>
+  <img
+    src="/img/robot_images/husky_images/robot_husky_18.png"
+    width="600"
+  />
+</center>
+
+In general, following these tips will help maximize the life of the battery:
+
+- Always fully charge the battery as soon as you are finished using it.
+- Charge batteries at room temperature. Never charge lead-acid batteries at temperatures above 35 degrees Celcius.
+- Charge batteries in a well-ventilated area.
+- Do not allow the battery to freeze. A low battery will freeze sooner than a fully charged one. Never charge a frozen battery.
+- Stored batteries should be topped up every month. Completely cycle batteries every 4-6 months.
+- At a discharge rate of 50% DOD (depth of discharge), the lifecycle of the Husky battery is 450-550 cycles.
+  After this point, reduced capacity will be apparent.
+- Regularly discharging the battery below 50% DOD will reduce the lifecycle of the battery, sometimes to less than 300 cycles.
+
+## Lithium Ion Battery Care
+
+If your Husky has an upgrade to Lithium Ion batteries, note the required maintenance below.
+
+### Balancing Lithium Ion Batteries
+
+The UGV's batteries must be balanced on a regular basis to ensure accurate
+feedback, consistent performance, and long life. If the batteries are
+allowed to fall out of balance, they will become unevenly loaded. This can
+lead to back-feeding between batteries, reduced charge-holding capacity
+for the system as a whole, and increased stress on both batteries.
+	
+If the imbalance becomes severe enough, one of the system batteries
+will enter a protective state and disconnect itself from the system.
+Once the battery enters this state there is a limited time window to
+recover it. If that window expires the battery will become irreversibly
+damaged with no possibility of recovery.
+
+Battery balancing is a slow process that will occur automatically when
+the fully-charged batteries are held at their float voltage. Because the
+automatic charger will shut off when it determines that the batteries are
+fully charged, the system cannot be balanced using the autocharge dock.
+The manual charger must be used.
+
+To perform a "quick" balance, leave the robot plugged into the manual
+charger overnight. A quick balance is recommended at least once per week.
+A full balance of 24+ hours is recommended at least once per month.
+
+Contact Clearpath Robotics for assistance with recovering a battery that
+has entered a protective state.
+
+### Checking Lithium Ion Battery Health
+
+A regularly-balanced battery system can be recharged thousands of times
+before degrading. Eventually, however, the batteries will degrade (their
```

Comment:
This sentence is hard to understand:

- _Eventually, however, the batteries will degrade (their usable capacity will decrease)_

Consider changing it to:

- _Even with good battery maintenance, the battery will loose capcity over time. This results in reduced runtime and driving range for your Husky._

Reply:
Updated